.codefresh/ui-build-deploy.yaml

```diff
@@ -10,6 +10,13 @@ steps:
     stage: clone
     repo: atk4/ui
     revision: "${{CF_BRANCH}}"
+
+  fix_sha:
+    stage: clone
+    title: Inject dependency SHA to bypass cache
+    commands:
+      - ./tools/inject-dependency-sha.sh
```

Comment:
I think it is still better to do in DockerFile, as it can be run also locally.

Reply:
if files are produced on a separate step, it allows us to cache docker layers more efficiently when dependencies haven't been updated.

Reply:
Really? composer install can not be cached, so if this is after it...